src/en/news/blog/README.md

```diff
@@ -9,12 +9,11 @@
 title: Blog post title
 date: yyyy-mm-dd
 author: Blog post author name
-image: "/assets/image.jpg"
+image: "image.jpg"
```

Comment:
Should this be...?

```suggestion
image: "images/image.jpg"
```

Reply:
Yes. I could have sworn updated that.

I've just checked my local and noticed I hadn't saved the file, doh! 🙍‍♂️